src/engine/types/sequence.ts

```diff
@@ -264,7 +264,7 @@ export class Sequence extends Type<SequenceDefinition> {
     }
 
     ISPLAYING() {
-        throw new NotImplementedError()
+        return this.sequenceName !== null
```

Comment:
Will sequenceName be null after sequence ends by itself? STOP() is not called when sequence ends

Reply:
Good catch, fixed that in `progressNext` (L159) just now.

Reply:
This fix broke kogut in S14_0_PRZEDKOGUTEM Ufo

Reply:
![ohno](https://github.com/user-attachments/assets/9be33a16-b3ec-4d6e-aa6e-458109dbe87d)

Reply:
Yeah, putting it *before* ONFINISHED is definitely a better idea.